lib/core/src/packStruct.cpp

```diff
@@ -3391,8 +3393,12 @@ int packStruct(const void *inStruct,
 
         /* add a NULL termination */
         *static_cast<char*>(outPtr) = '\0';
-        if ( getRodsLogLevel() >= LOG_DEBUG9 ) {
-            printf( "packed XML: \n%s\n", ( char * ) packedOutput.bBuf.buf );
+        if (getRodsLogLevel() >= LOG_DEBUG9) {
```

Comment:
there are only two LOG_DEBUG9's in this diff - the rest are LOG_DEBUG8.

not sure... if that's important.

Reply:
According to the docs, a log level of 8 causes the wireline XML packing instructions to be logged.

My guess is that LOG_DEBUG9 is a typo and is meant to be LOG_DEBUG8. Regardless, people outside of the core dev team have been able to implement the XML protocol without changing this.

Overall, it might not be that big of a deal to change it for 4.3.0.

Reply:
Seems fine - just didn't see an obvious reason for the difference.   Happy to leave it alone.

Reply:
I'm going to leave it as is.